docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-promotions-discounts-feature-integration.md

```diff
@@ -35,44 +35,18 @@ Make sure that the following modules were installed:
 
 | MODULE | EXPECTED DIRECTORY |
 | - | - |
-| DiscountMerchantSalesOrder    | spryker/discount-merchant-sales-order     |
-| DiscountMerchantSalesOrderGui | spryker/discount-merchant-sales-order-gui |
+| DiscountMerchantSalesOrder    | vendor/spryker/discount-merchant-sales-order     |
+| DiscountMerchantSalesOrderGui | vendor/spryker/discount-merchant-sales-order-gui |
 
 {% endinfo_block %}
 
-### 2) Set up the transfer objects
+### 2) Set up configuration
 
-Generate transfer changes:
-
-```bash
-console transfer:generate
-```
-
-{% info_block warningBox "Verification" %}
-
-Make sure that the following changes were applied in transfer objects:
-
-| TRANSFER  | TYPE  | EVENT | PATH |
-| - | - | - | - |
-| MerchantOrder.merchantOrderItems | attribute | created | src/Generated/Shared/Transfer/MerchantOrderTransfer |
-
-{% endinfo_block %}
-
-### 3) Zed translations
-
-Generate a new translation cache for Zed:
-
-```bash
-console translator:generate-cache
```

Comment:
why removed?

Reply:
not removed, just placed in order as in IG guides.